### ISSUE TITLE
Support EXT_meshopt_compression in useGLTF

### ISSUE THREAD

**Issue by mattrossman** [pmndrs/drei#241]
### Describe the feature you'd like:

I'd like to support the `EXT_meshopt_compression` glTF extension in the `useGLTF` hook, which enables further file compression beyond what Draco can achieve. See: https://www.npmjs.com/package/gltfpack

For example, with Draco alone I can shrink a 600 KB model down to 430 KB. With `gltfpack` using the `-c` flag I can shrink it all the way down to **130 KB**.

### Suggested implementation:

Supporting this involves:
```js
import { MeshoptDecoder } from 'three/examples/jsm/libs/meshopt_decoder.module'

// In the draco function
loader.setMeshoptDecoder(MeshoptDecoder)
```

I'm not sure if there's any disadvantage to always leaving this on, but if needed we could add another parameter after the draco path to allow users to customize the behavior.

**Comment by joshuaellis** [comment]
Hi @mattrossman,

I've just looked at the impl for `useGLTF` we use `DRACOLoader` from [three](https://github.com/mrdoob/three.js/blob/master/examples/jsm/loaders/DRACOLoader.js) unfortunately, I can't see a method called `setMeshoptDecoder` maybe i've misunderstood? 

It might be easier & quicker for you to create a PR. But i'm concerned this needs to be done on `three` not here?

**Comment by mattrossman** [comment]
Done. Btw the loader I was referring to is the `GLTFLoader` passed to the function that `draco` returns.

**Comment by joshuaellis** [comment]
Ah I see, thanks @mattrossman for that, this has been released in `2.3.0-beta.5`
